huaweicloud/resource_huaweicloud_dcs_instance_v1.go

```diff
@@ -148,6 +148,18 @@ func resourceDcsInstanceV1() *schema.Resource {
 					},
 				},
 			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"enterprise_project_name": {
```

Comment:
suggest to only use `enterprise_project_id` to keep consistent with other resources.